primitives/primitives/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+authors = ['ChainSafe Systems']
+description = 'Primitive types and traits for PINT.'
+edition = '2018'
+license = 'LGPL-3.0-only'
+name = 'primitives'
+readme = 'README.md'
+repository = 'https://github.com/ChainSafe/PINT/'
+version = '0.0.1'
+
+[dependencies]
+serde = { version = "1.0.101", features = ["derive"], optional = true }
+codec = { package = 'parity-scale-codec', version = '2.0.0', default-features = false, features = ['derive'] }
+frame-support = { git = 'https://github.com/paritytech/substrate', branch = 'polkadot-v0.9.4', default-features = false }
+frame-system = { git = 'https://github.com/paritytech/substrate', branch = 'polkadot-v0.9.4', default-features = false }
+
+# Polkadot Dependencies
+xcm = { git = 'https://github.com/paritytech/polkadot', branch = 'release-v0.9.4' }
```

Comment:
> The test error does not make any sense to me...
> any ideas @clearloop ? 🤔

LOL, these seem caused by `[features.std]` in `Cargo.toml`

add `xcm/std` to `[features.std]` will solve this

Reply:
substrate dependencies must be marked `std`, because these are required by the build-script of runtime, 

* Compile substrate project
  * compile runtime to wasm ( generate template `Cargo.toml`s )
  * re-compile the project

Reply:
ofc a missing `std`...
awesome!

Reply:
`xcm = { git = 'https://github.com/paritytech/polkadot', branch = 'release-v0.9.4', default-features = false }`